src/roiextractors/extractors/inscopixextractors/inscopixsegmentationextractor.py

```diff
@@ -97,3 +228,167 @@ def get_sampling_frequency(self) -> float:
             return 1 / self.cell_set.timing.period.secs_float
         except AttributeError:
             return None
+
+    def get_session_start_time(self) -> Optional[datetime]:
```

Comment:
```suggestion
    def get_session_start_time(self) -> datetime | None:
```
You can also write it like this which for output I think it is more clear.
What do you think?